RELEASING.md

```diff
@@ -122,36 +155,61 @@ Before any artifacts can be generated and vote can be made the version number wi
 
 #### Update release version
 
-This command will prompt you for the release info and create a commit that updates the version from `x.y.z-SNAPSHOT` to `x.y.z`.
-This will also update the SCM tag.
-The current `pom.xml` files will be backed up with the name `pom.xml.releaseBackup` and left in staging.
+This command will remove `SNAPSHOT` from all versions in `pom` files.
 
 ```Shell
-mvn clean release:prepare
+mvn versions:set -DremoveSnapshot=true -DgenerateBackupPoms=false
 ```
 
-#### Create the Pull Request
+After that open the main `pom.xml` and change `<tag>HEAD</tag>` to the necessary release tag.
```

Comment:
After the version has been set, update the scm tag by opening the main `pom.xml` and changing `<tag>HEAD</tag>` to match the release tag.